src/dotnet/APIView/APIViewWeb/Client/src/comments.ts

```diff
@@ -90,6 +90,18 @@
         e.preventDefault();
     });
 
+    $(document).on("keydown", ".new-thread-comment-text", e => {
+        if (e.ctrlKey && e.keyCode === 13) {
+            const form = $(e.target).closest("form");
+            if (form) {
+                const button = form.children("[data-post-update='comments']");
```

Comment:
Ugh, I wonder if we can do something nicer. What about extracting the original `data-post-update` handler into a method and calling it from both places?

Reply:
What about just replacing the form.submit handler?

Reply:
Try it, might look good.